dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyForNonFinancialsPanel.vue

```diff
@@ -154,7 +145,7 @@ export default defineComponent({
      * @returns the resulting string
      */
     formatPercentageNumber(relativeShareInPercent: number) {
-      return `${relativeShareInPercent.toFixed(2).toString()} %`;
+      return `${relativeShareInPercent.toFixed(2).toString()}`;
```

Comment:
Why no % anymore? Where does the % in the view page come from? In any case: The Documentation is wrong now.

Reply:
Will be standardized by the formatPercentageNumberAsString function which will add the % itself

Reply:
And function was extracted to a util toolbox

Reply:
![grafik](https://github.com/d-fine/Dataland/assets/71646830/b8a4e6fb-cec8-46ce-862d-0c5e37442fb3)

Reply:
This seems wrong

Reply:
We have a subtask to fix this. @MarcinStanczyk7 is working on it

Reply:
fixed